docs/howto-write-exercises.md

```diff
@@ -60,7 +60,7 @@ specific shape, illustrated by the following ascii art:
 ```
 
 The complete format specification for exercise description is given in
-[https://github.com/ocaml-sf/learn-ocaml/blob/master/docs/exercices_format.md](learn-ocaml/docs/exercices_format.md).
+[docs/exercices_format.md](exercices_format.md).
```

Comment:
I would prefer an absolute link because we do not know from where this file is read.